PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -1006,7 +1145,14 @@ public DebugScenario GetDebugScenario()
             }
             else if (_runspace.ConnectionInfo is WSManConnectionInfo)
             {
-                return DebugScenario.RemoteSession;
+                if (_runspace.RunspaceAvailability == RunspaceAvailability.RemoteDebug)
+                {
+                    return DebugScenario.RemoteAttach;
+                }
+                else
+                {
+                    return DebugScenario.RemoteSession;
+                }
             }
             else if (_runspace.ConnectionInfo != null && !(_runspace.ConnectionInfo is WSManConnectionInfo))

```

Comment:
Agreed. But I think specifying the not null will make it super clear to anyone in the future what the criteria is.